tools/tinify_images.py

```diff
@@ -20,8 +33,8 @@ def tinify_images(path, extensions):
     parser.add_argument('-k', '--key', type=str, required=True,
                         help='API key')
 
-    parser.add_argument('-p', '--path', type=str, required=True,
```

Comment:
Please don't change existing argument names. Old saved commands should still work.